app/electron/main.ts

```diff
@@ -202,7 +205,7 @@ app.on('quit', function () {
     } else {
       intentionalQuit = true;
       log.info('stopping server process...');
-      serverProcess.stdin.pause();
+      serverProcess.stdin.destroy();
```

Comment:
Can you explain this change? Doesn't seem related to the port.

Reply:
so no pause method exists on stdin(type Writable)(typescript helped to get this) we want to use destory instead